engine/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsController.mm

```diff
@@ -576,8 +583,21 @@ - (void)applyMutators:(const flutter::MutatorsStack&)mutatorsStack
         CGFloat blurRadius = (*iter)->GetFilterMutation().GetFilter().asBlur()->sigma_x();
         UIVisualEffectView* visualEffectView = [[UIVisualEffectView alloc]
             initWithEffect:[UIBlurEffect effectWithStyle:UIBlurEffectStyleLight]];
+
+        // TODO: Technically we should be using the largest corner radius, but currently
```

Comment:
this should be tracked by a github issue

Reply:
Filed as https://github.com/flutter/flutter/issues/179126